logical/framework/openapi_test.go

```diff
@@ -10,6 +11,8 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/hashicorp/vault/helper/wrapping"
+
```

Comment:
Would be nice to strip this extra space.

Reply:
Done